### DISCUSSION TITLE
After migrating to v2 it became impossible to work with wagmi, please give someone a normal code example with web3-react

### DISCUSSION THREAD

**Discussion by DevTeaLeaf** [WalletConnect/walletconnect-monorepo#2935] [Developer Support]
In the documentation, the process for switching to web3-react is described in 2 words, I searched all over the Internet for what to do next with those code snippets, but I did not find it

**Comment by crypblizz8** [comment]
Hey there, thanks for your feedback: here is our web3-react migration guide: https://docs.walletconnect.com/2.0/advanced/migration-from-v1.x/dapps#web3-react

Is there something missing here you would like us to improve?

**Reply by DevTeaLeaf** [reply]
The problem is that it doesn’t show how to script it using web3-react, that is, in which component to wrap the App and so on, but I didn’t find it on the Internet, can you have an example of a whole small web3-react application?
